src/store/index.ts

```diff
@@ -1,17 +1,19 @@
 import {StoreOptions} from 'vuex'
 import {IFlowsState, store as flow} from './flow'
 import {IBuilderState, store as builder} from './builder'
-
+import trees from './trees/trees'
 
 export interface IRootState {
   builder: IBuilderState,
   flow: IFlowsState,
+  trees: any
```

Comment:
The trees store is loaded from InteractionDesigner : https://github.com/FLOIP/flow-builder/blob/master/src/views/InteractionDesigner.vue#L167, not only for stories. Therefore, a block editor component loaded outside this InteractionDesigner will be broken. That's why I tried to add this.
I have a suggestion: What if instead of loading the `trees` store from InteractionDesigner, we put it here (as I added)? (meaning, I need to change the store import in InteractionDesigner to make things uniform)

Reply:
Implemented this:
- remove some imports on InteractionDesigner: https://github.com/FLOIP/flow-builder/pull/12/files#diff-4778731e3debd774a6c14025179b9bfb34aae3d45fb42448205377ca6475503dL59
- then import the main `store` : https://github.com/FLOIP/flow-builder/pull/12/files#diff-4778731e3debd774a6c14025179b9bfb34aae3d45fb42448205377ca6475503dR57

- And change the module definition here: https://github.com/FLOIP/flow-builder/pull/12/files#diff-4778731e3debd774a6c14025179b9bfb34aae3d45fb42448205377ca6475503dR163

What do you think ?

Reply:
I think this is a definite improvement. @bzabos Do you forsee any problems with this?